Source/SwiftLintBuiltInRules/Rules/Style/SwitchCaseAlignmentRule.swift

```diff
@@ -216,5 +227,46 @@ extension SwitchCaseAlignmentRule {
                 """)
             ]
         }
+
+        private var validOneLiners: [Example] = [
+            Example(
+                "switch i { case .x: 1 default: 0 }",
+                configuration: ["ignore_one_liners": true]
+            ),
+            Example(
+                "let a = switch i { case .x: 1 default: 0 }",
+                configuration: ["ignore_one_liners": true]
+            )
+        ]
+
+        private var invalidOneLiners: [Example] {
+            [
+                // Default configuration should not ignore one liners
+                Example(
+                    "switch i { \(violationMarker)case .x: 1 \(violationMarker)default: 0 }"
+                ),
+                Example("""
+                switch i {
+                case .x: 1 default: 0 }
+                """, configuration: ["ignore_one_liners": true]),
+                Example("""
+                switch i { case .x: 1 default: 0
+                }
+                """, configuration: ["ignore_one_liners": true]),
+                Example("""
+                switch i
+                { case .x: 1 default: 0 }
+                """, configuration: ["ignore_one_liners": true]),
+                Example("""
+                let a = switch i {
+                case .x: 1 default: 0
+                }
+                """, configuration: ["ignore_one_liners": true]),
+                Example("""
+                let a = switch i {
+                case .x: 1 default: 0 }
+                """, configuration: ["ignore_one_liners": true])
```

Comment:
Good examples! They should have violation markers, though.

Reply:
Fair point 😁